angularjs-portal-frame/src/main/webapp/partials/search.html

```diff
@@ -1,6 +1,7 @@
 <form ng-submit="submit()">
   <div class="input-group">
-    <input type="search" class="main-search" placeholder="Search MyUW" name="initialFilter" ng-model="initialFilter" aria-label="Search MyUW" focus-me="showSearchFocus">
+    <input type="search" class="main-search" placeholder="Search MyUW" name="initialFilter" ng-model="initialFilter" aria-label="Search MyUW" focus-me="showSearchFocus"
+            typeahead="title as match.title for match in filterMatches | limitTo:10" typeahead-on-select="onSelect($item, $model, $label)" typeahead-loading="portletListLoading" typeahead-min-length="2">
```

Comment:
Interesting thing that came up in review. It may be interesting to have a different search for autocomplete that only searched on the name/title.